<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeDlg</class>
 <widget class="QDialog" name="NodeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1" colspan="3">
    <widget class="QLineEdit" name="linePath"/>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="lineId">
     <property name="toolTip">
      <string>Name</string>
     </property>
     <property name="placeholderText">
      <string>Path/ID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelId">
     <property name="text">
      <string>ID/Sig</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushType">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="ColorButton" name="clrNode">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QPushButton" name="pushCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>108</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QTextEdit" name="textText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Description</string>
     </property>
     <property name="placeholderText">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="pushOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="lineName">
     <property name="toolTip">
      <string>Title (user text)</string>
     </property>
     <property name="placeholderText">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelPath">
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="pushOverview">
     <property name="text">
      <string>Overview...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>../src/gui/widgets/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineId</tabstop>
  <tabstop>lineName</tabstop>
  <tabstop>textText</tabstop>
  <tabstop>pushType</tabstop>
  <tabstop>pushOk</tabstop>
  <tabstop>pushCancel</tabstop>
  <tabstop>clrNode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
